hw1/02-palindrome.js

```diff
@@ -1,3 +1,58 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", handleInput);
+
+function handleInput(elem) {
+  ///check for negative input
+  if (elem.target.value < 0) {
```

Comment:
I think that if you have more time, we could check for more scenarios of user inputs.
For example the user can still enter non-number like `e` or `+`.  For example user input: `12+21` or `111e2`.

Additionally, when there is no longer any user input, we should display a proper message.  For example, when the user types 1 and then backspace to delete all the input the web page displays `Yes, this is a palindrome!`.